### ISSUE TITLE
Add Biofiltration Swale Lines to mobile map

### ISSUE THREAD

**Issue by ReedHunter-Wsdot** [WSDOT/HATS#281]
Biofiltration Swale, a subtype of ditches, is available as lines in Hats Web, but has been a point type in Hats iOS since 1.0.  Nobody has complained, but this needs to be supported as a line type in 1.4.  This is a schema change, so it will require everyone to redownload a new offline map.  There are only 23 features so far in Hats Web missing from Hats iOS, and there has been so far only 1 Biofiltration point added in Hats iOS.  For now if folks need to enter a Biofiltration Swale they can enter a point feature.

**Comment by tangoai** [comment]
Reed, Biofiltration Swales are linear but I don't believe Bio-Infiltration Swales are in HATS IOS. Trett

**Comment by ReedHunter-Wsdot** [comment]
I should have called it Wet Biofiltration Swale.  RoadsideFeatureTypeId of 83, ParentFeatureTypeId of 9.

**Comment by ReedHunter-Wsdot** [comment]
Josh, I'm adding Wet Biofiltration Swales at the same time that I'm adding the several new feature types.  Should it matter that there are two rows in dbo.LocalFeatureType for this feature type?  I notice that the FeatureTypeActiveFlag is 0 for one of them.  I know that we talked about this before, but I don't remember the answer, which is why I'm asking in Github.  Thanks

**Comment by jhudsonWA** [comment]
As long as only one is active that is all that maters. We can now tag the other as “duplicate” just to make sure it doesn’t get turned on. I’ve seen other issues like this where the feature was added as a non-inventory feature and then when added to the inventory it was added again but has it’s roadsidefeaturetypeid set. But too much info.. as long as we have an active flag on one that’s all that gets synced.

On Mar 21, 2016, at 5:03 PM, Reed Hunter <notifications@github.com<mailto:notifications@github.com>> wrote:

Josh, I'm adding Wet Biofiltration Swales at the same time that I'm adding the several new feature types. Should it matter that there are two rows in dbo.LocalFeatureType for this feature type? I notice that the FeatureTypeActiveFlag is 0 for one of them. I know that we talked about this before, but I don't remember the answer, which is why I'm asking in Github. Thanks
